modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -168,7 +168,7 @@ def test_read_single_csv_with_parse_dates(self, parse_dates):
     Engine.get() != "Ray", reason="Currently only support Ray engine for glob paths."
 )
 def test_read_multiple_csv_s3():
-    path = "S3://modin-datasets/testing/multiple_csv/test_data*.csv"
+    path = "s3://modin-datasets/testing/multiple_csv/test_data*.csv"
```

Comment:
Why was this changed?

Reply:
It is simpler for us to not deal with malformed protocols in the paths and this is what `fsspec` decided to do.

Reply:
For more context: https://github.com/fsspec/filesystem_spec/issues/1009